### DISCUSSION TITLE
Python模块化开发

### DISCUSSION THREAD

**Discussion by AkimioJR** [whyour/qinglong#2284] [Q&A]
我有一个python项目刚好想放在青龙面板上跑，由于采用了模块化开发，有多个py文件，但是只有一个main.py为入口，使用git拉库的时候会把所有的模块文件都加入定时任务中，但是这些模块文件都不能运行，会导致定时任务看起来乱乱的，有没有什么好的方法

**Comment by whyour** [comment]
你白名单只加一个 main.py 就会只添加一个 定时任务
